src/pages/Home.js

```diff
@@ -23,27 +23,30 @@ const Home = () => {
   const [selectedCoin, setSelectedCoin] = useState(null);
   const [addToWatchlist, setAddToWatchlist] = useState('');
   const [query, setQuery] = useState('');
+  const [loggedIn, setLoggedIn] = useState(false)
+  const [userUID, setUserUID] = useState('')
 
   const { handleAllCoins } = useApi();
 
   const onSearch = (value) => getQueryCoins(value)
     .then(res => setQuery(res.coins.map(coin => coin.id).join(", ")));
 
+    //uncomment to use saved sampleData
   useEffect(() => {
-    // handleAllCoins().then((res) => {
       setCoins(sampleData);
       setSelectedCoin(sampleData[0]);
       setIsLoading(false);
-    // });
   }, []);
 
-  useEffect(() => {
-    setIsLoading(true);
-    getQueryCoinsInfo(query).then((res) => {
-      setCoins(res);
-      setIsLoading(false);
-    })
-  }, [query])
+  // uncomment to use live data
+  // useEffect(() => {
+  //   setIsLoading(true);
+  //   getQueryCoinsInfo(query).then((res) => {
+  //     console.log(res)
+  //     setCoins(res);
+  //     setIsLoading(false);
+  //   })
+  // }, [query])
```

Comment:
`.then` can accept two methods as an argument one for `resolved` and the other for `reject`. You can add the commented method as the second argument.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then

Reply:
This is a comment for @timothyrusso ! I just switched between using the API and using the sample data.

Reply:
conflicts resolved I think. I left the useEffect as they were in the main/master and just inserted what I changed. 

Also changed the sample data to just include the 100 coins I had put in. Let me know if ok to resolve

Reply:
I guess that's better.